### ISSUE TITLE
Refactor to Clean Architecture

### ISSUE THREAD

**Issue by shibanovp** [experienced-dev/java-starter-template#12]
e.g. https://blog.sourced-bvba.be/article/2020/03/11/ca-quarkus/

**Comment by shibanovp** [comment]
I am not sure what is the best practice in Spring.
Is it relevant?

https://www.baeldung.com/spring-boot-clean-architecture
https://www.youtube.com/watch?v=mbNzUkNjrnA

**Comment by VishovVladimir** [comment]
How can we implement it now? we don't have any  logic in our app

**Comment by shibanovp** [comment]
> We don't have any logic in our app.

Yes, we don't have any Enterprise Business Rules or Application Business Rules. We only have a framework. Do you think it makes sense to reorganize the code (e.g., having a "framework" folder) if we want the ability to deploy the same codebase to different environments, such as:
* AWS Lambda
* EKS
* Azure Functions
* AKS
* GCP Cloud Functions
* GKE?

**Comment by VishovVladimir** [comment]
I shure that we will not have problem with code base during depoy
